src/org/loklak/harvester/TwitterScraper.java

```diff
@@ -26,25 +26,17 @@
 import org.loklak.objects.AbstractObjectEntry;
 import java.net.MalformedURLException;
 import java.net.URL;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Date;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
see this https://docs.oracle.com/javase/7/docs/api/java/util/package-summary.html